client/my-sites/posts/post-list.jsx

```diff
@@ -301,4 +301,9 @@ var Posts = React.createClass( {
 	}
 } );
 
-module.exports = PostList;
+export default connect(
+	( state ) => ( {
+		selectedSiteId: getSelectedSiteId( state ),
+		hasSites: !! getSites( state ).length
```

Comment:
> This will calculate all sites calling `getSite` for all of them, which is unnecessary for just counting the sites. 

Yeah, good point.

> Maybe we should add a separate selector for this one? Should be a pretty straightforward one.

I'm a bit reluctant, since it'd be the only use case, and mostly to replace a `sites-list` property that doesn't have a perfectly corresponding Redux counterpart (see [previous](https://github.com/Automattic/wp-calypso/pull/13121#discussion_r111932935) [discussions](https://github.com/Automattic/wp-calypso/pull/13121#discussion_r111933117)).

I might actually do an `! isEmpty( state.sites.items )`.